blog/_posts/2017-07-04-pivotal-london-intro.md

```diff
@@ -0,0 +1,57 @@
+---
+title: And so it begins!
+class: page blog
+layout: bootstrap
+current: blog
+permalink: blog/2017-07-04-and-so-it-begins
+author: Emma Deacon and Kara de la Marck
+twitter: EmmaMDeacon KaraMarck
+categories:
+- student-posts
+
+---
+
+
+
+Hi, we’re [Emma](https://twitter.com/EmmaMDeacon) and [Kara](https://twitter.com/KaraMarck), the members of Pivotal, London! We’re working on [Babel](https://github.com/babel/babel) the JavaScript language compiler, with the help of our incredible mentor [Henry Zhu](https://twitter.com/left_pad), our supportive supervisor [Ines](https://twitter.com/ines_opcoelho), and our amazing hosts [Pivotal](https://twitter.com/pivotal). We’re so lucky to be in this position, and we cannot thank Rails Girls Summer of Code enough!!
+
+![Alt Text](https://media.giphy.com/media/k3kqJ2d8cUvSM/giphy.gif)
```

Comment:
As it says in the [contributing guidelines](https://github.com/rails-girls-summer-of-code/summer-of-code/blob/gh-pages/CONTRIBUTING.md#images), we need to have a credit for the images / link to a source for all images used. Could you please still add that? You can find examples in past blog posts, but feel free to ping me directly if you need more clarifications.